### ISSUE TITLE
Fix accessibility testing to include slow internet connection

### ISSUE THREAD

**Issue by kelliedesigner** [barnardos/design-system#853]
Accessibility testing page is missing guidance on testing for the slow internet connection.

**Comment by kelliedesigner** [comment]
Key findings from [Ofcom](https://www.ispreview.co.uk/index.php/2018/05/ofcom-2018-study-average-home-broadband-speeds-hit-46-2mbps.html):

- a little under half of the country still use the much slower and less reliable ADSL based pure copper lines
- Wales suffers for much slower internet speeds than England
- A large gap in speeds between urban and rural areas persists
- Network congestion, poor home wiring, faults and traffic management measures can also have an impact on internet speed
- not everyone can afford, is aware of, or feels the need to upgrade a faster connection, even when one may be available.
- there are still remote rural communities and digitally disadvantaged urban pockets who don't have better options available

- 2% of the UK's population (11% of the UK's rural population) has a speed of 2MBPS or less
- ADSL network speeds can be as slow as 3.8MBPS

![ofcom_2018_broadband_speeds_by_technology](https://user-images.githubusercontent.com/6122118/58171257-7a8ffb00-7c8d-11e9-9402-4cfe4962bb1a.png)

**Comment by kelliedesigner** [comment]
Closing as Lighthouse includes latency checks
